en/docs/install-and-setup/setup/si-setup/si-cicd-k8s.md

```diff
@@ -0,0 +1,190 @@
+
+[![K8s based CI/CD for Streaming Integrator]({{base_path}}/assets/img/deploy/si-cicd-k8s.png)]({{base_path}}/assets/img/deploy/mi-cicd-k8s.png)
+
+#### Deployment Descriptor Repository
+- This repository will be used to track the Siddhi apps/deployments in each environment.
+- We maintain the Deployment Descriptor repository in GitHub. 
+- There are Kubernetes YAML files for each project inside a separate folder.
+- This repository will have three branches representing each deployment environment.
+- To use a new version / rollback to a previous version, the user should define the version inside the respective `deployment.yaml` and commit to the respective branch.
+
+#### Deployment Descriptor Jenkins Build Job
+- Maintain one Jenkins job per Environment.
+- This job contains only the build phase. This will apply the relevant YAML files in the relevant environment.
+
+### Kubernetes artifacts
```

Comment:
is there a place we say recommended si server resources ? there shall we add the recommended ram and cpu

Reply:
Also lets mention this is a reference guide for active active deployment where we can scale horizontally and we can point to 4.0.0 guide of this content for other deployment patterns https://ei.docs.wso2.com/en/latest/streaming-integrator/setup/deploying-si-as-an-active-active-deployment/

Reply:
Regarding the si server resources, the recommended ram and CPU is given in the shared deployment.yaml.

Reply:
I mentioned the deployment patterns as you requested. Please review